editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -728,7 +728,7 @@ void EditorAssetLibrary::_image_update(bool use_cache, bool final, const PoolByt
 		if (r.ptr()) {
 			if (memcmp(&r[0], &png_signature[0], 8) == 0) {
 				image->copy_internals_from(Image::_png_mem_loader_func(r.ptr(), len));
-			} else if (memcmp(&r[0], &jpg_signature[0], 3) == 0) {
+			} else if ((memcmp(&r[0], &jpg_signature[0], 3) == 0) && Image::_jpg_mem_loader_func) {
```

Comment:
Might be worth adding the same check for PNG. It's not optional right now and unlikely to become, but who knows :)